packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -56,6 +58,8 @@ export default createConnector({
     limitMax: PropTypes.number,
     defaultRefinement: PropTypes.string,
     transformItems: PropTypes.func,
+    withSearchBox: PropTypes.bool,
+    searchForFacetValues: PropTypes.bool, //deprecated
```

Comment:
You can use @deprecated so we can easily search for those tokens for a new major version